compiler/next/include/chpl/resolution/scope-types.h

```diff
@@ -248,63 +251,120 @@ class Scope {
   };
-  Kind kind = SYMBOL_ONLY;
-  bool isPrivate = true;
+
+private:
+  ID symbolId_;      // ID of the imported symbol, e.g. ID of a Module
+  Kind kind_ = SYMBOL_ONLY;
+  bool isPrivate_ = true;
 
   // the names/renames:
   //  pair.first is the name as declared
   //  pair.second is the name here
-  std::vector<std::pair<UniqueString,UniqueString>> names;
+  std::vector<std::pair<UniqueString,UniqueString>> names_;
 
+public:
   VisibilitySymbols() { }
   VisibilitySymbols(ID symbolId, Kind kind, bool isPrivate,
                     std::vector<std::pair<UniqueString,UniqueString>> names)
-    : symbolId(symbolId), kind(kind), isPrivate(isPrivate),
-      names(std::move(names))
+    : symbolId_(symbolId), kind_(kind), isPrivate_(isPrivate),
+      names_(std::move(names))
   { }
 
+  /** Return the ID of the imported symbol, e.g. ID of a Module */
+  const ID &symbolId() const { return symbolId_; }
+
+  /** Return the kind of the imported symbol */
+  Kind kind() const { return kind_; }
+
+  /** Return whether or not the imported symbol is private */
+  bool isPrivate() const { return isPrivate_; }
+
+  /** Lookup the declared name for a given name
+      Returns true if `name` is found in the list of renamed names and
+      stores the declared name in `declared`
+      Returns false if `name` is not found
+  */
+  bool lookupName(const UniqueString &name, UniqueString &declared) const {
+    for (const auto &p : names_) {
+      if (p.second == name) {
+        declared = p.first;
+        return true;
+      }
+    }
+    return false;
+  }
 
-  bool operator==(const VisibilitySymbols& other) const {
-    return symbolId == other.symbolId &&
-           kind == other.kind &&
-           names == other.names;
+  bool operator==(const VisibilitySymbols &other) const {
+    return symbolId_ == other.symbolId_ &&
+           kind_ == other.kind_ &&
+           names_ == other.names_;
   }
   bool operator!=(const VisibilitySymbols& other) const {
     return !(*this == other);
   }
 
   void swap(VisibilitySymbols& other) {
-    symbolId.swap(other.symbolId);
-    std::swap(kind, other.kind);
-    names.swap(other.names);
+    symbolId_.swap(other.symbolId_);
+    std::swap(kind_, other.kind_);
+    names_.swap(other.names_);
   }
 };
 
-// Stores the result of in-order resolution of use/import
-// statements. This would not be separate from resolving variables
+/**
+ Stores the result of in-order resolution of use/import
+ statements.
+*/
+// This would not be separate from resolving variables
 // if the language design was that symbols available due to use/import
 // are only available after that statement (and in that case this analysis
 // could fold into the logic about variable declarations).
-struct ResolvedVisibilityScope {
-  const Scope* scope;
-  std::vector<VisibilitySymbols> visibilityClauses;
+class ResolvedVisibilityScope {
+private:
+  const Scope* scope_;
+  std::vector<VisibilitySymbols> visibilityClauses_;
+
+public:
```

Comment:
Do you think we should
 a) allow a mix of indentation style for public/private in class bodies
 b) never indent them (like here)
 c) indent them with one space (like other compiler/next code I've been writing)
?

I am curious what you think about it but it should not hold up this PR.

Reply:
I'd rather have one style and I've adjusted to match the existing one. We could look at using `clang-format` everywhere to all be on the same page. The right configuration var is `AccessModifierOffset -1`

Reply:
FWIW I think that clang-format would be a great practice for us to get in to but I havn't looked at using it myself. I don't have any idea of the difficulty / level of effort to using it.